__tests__/jestGlobalSetup.ts

```diff
@@ -0,0 +1,255 @@
+/* eslint-disable no-console */
+/**
+ * Asynchronous Global Test Setup
+ * Run only once
+ * ref: order of execution jestGlobalSetup.ts -> jest.setup.ts -> fixtures.ts
+ */
+
+import { BaseUrl } from '../src/constants';
+
+type DevnetStrategy = {
+  isDevnet: boolean;
+  isRS: boolean;
+};
+type ProviderType = {
+  sequencer: boolean;
+  rpc: boolean;
+};
+
+/**
+ * Global Setup Fixtures
+ */
+
+/* Default test config based on run `starknet-devnet --seed 0` */
+const GS_DEFAULT_TEST_PROVIDER_URL = 'http://127.0.0.1:5050/';
+
+const overrideOrSet = (envName: string, setValue: string) => {
+  process.env[envName] = process.env[envName] ?? setValue;
+};
```

Comment:
I would suggest incorporating the boolean string ternary assignments made in the calls to `overrideOrSet` within the method itself so the calls can be made less verbose. I would also suggest renaming the method without the `override` fragment. E.g.:


```suggestion
const asBooleanString = (b: boolean) => (b ? 'true' : 'false');
const setIfNullish = (envName: string, setValue: string | boolean | undefined) => {
  process.env[envName] ??= typeof setValue === 'boolean' ? asBooleanString(setValue) : setValue;
};
```

Reply:
Good sugestion, I updated it using nullish assignment with toString handling bools.